packages/lesmis-server/src/app/createApp.ts

```diff
@@ -7,15 +7,14 @@ import { TupaiaDatabase } from '@tupaia/database';
 import { OrchestratorApiBuilder, handleWith } from '@tupaia/server-boilerplate';
 import { LesmisSessionModel } from '../models';
 import { UserRoute, EntityRequest, EntityRoute, EntitiesRoute } from '../routes';
-import { verifyLogin } from '../auth';
+import { attachSession } from '../auth';
 
 /**
  * Set up express server with middleware,
  */
 export function createApp() {
-  return new OrchestratorApiBuilder(new TupaiaDatabase())
+  return new OrchestratorApiBuilder(new TupaiaDatabase(), attachSession)
     .useSessionModel(LesmisSessionModel)
-    .verifyLogin(verifyLogin)
```

Comment:
Removed verifyLogin check as we allow anyone to access the lesmis-server endpoints, whether they're logged in or not